### PR TITLE
12.0 mig account_chart_update

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: account_chart_update/wizard/wizard_chart_update.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 class WizardUpdateChartsAccounts(models.TransientModel):
     _name = 'wizard.update.charts.accounts'
+    _description = 'Update charts accounts'
```
Comment (MiquelRForgeFlow) [inline account_chart_update/wizard/wizard_chart_update.py]: I would name it `Wizard Update Charts Accounts` instead

File: account_chart_update/wizard/wizard_chart_update.py [hunk 2]
```diff
@@ -1030,6 +1014,7 @@ class WizardUpdateChartsAccountsFiscalPosition(models.TransientModel):
 
 class WizardMatching(models.TransientModel):
     _name = 'wizard.matching'
+    _description = "Matching order"
```
Comment (MiquelRForgeFlow) [inline account_chart_update/wizard/wizard_chart_update.py]: I would name it `Wizard Matching` instead

**Review by MiquelRForgeFlow — APPROVED** [review]
code review :+1:

File: account_chart_update/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2016 Jacques-Etienne Baudoux <je@bcim.be>
+# Copyright 2016 Sylvain Van Hoof <sylvain@okia.be>
+# Copyright 2015-2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Detect changes and update the Account Chart from a template",
+    "summary": "Wizard to update a company's account chart from a template",
```
Comment (victor-ditay) [inline account_chart_update/__manifest__.py:9]: Nothing that is blocking but by aesthetic would be better to use or simple quote or double quote but not mix

File: account_chart_update/wizard/wizard_chart_update_view.xml [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+                <field
+                    name="state"
+                    select="2"
+                    widget="statusbar"
+                    statusbar_visible="init,ready,done"
+                    statusbar_colors='{"ready":"blue","done":"blue"}' />
+            </header>
+            <group attrs="{'invisible':[('state','!=','init')]}">
+                <h3>
+                    <p><center>This wizard will update your accounts, taxes and fiscal positions according to the selected chart template</center></p>
```
Comment (victor-ditay) [inline account_chart_update/wizard/wizard_chart_update_view.xml]: Would be better indent this line if possible

File: account_chart_update/wizard/wizard_chart_update_view.xml [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+                            <field name="update_fiscal_position" />
+                        </group>
+                        <group string="Other options" attrs="{'invisible':[('state','!=','init')]}">
+                            <field name="continue_on_errors" />
+                            <field name="recreate_xml_ids" />
+                        </group>
+                    </group>
+                    <group>
+                        <h5>
+                            <p>If you leave these options set, the wizard will not just create new records, but also update records with changes (i.e. different tax amount)</p>
```
Comment (victor-ditay) [inline account_chart_update/wizard/wizard_chart_update_view.xml]: Would be better indent this line if possible

File: account_chart_update/wizard/wizard_chart_update_view.xml [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+                            <p>Note: Only the changed fields are updated.</p>
+                        </h5>
+                    </group>
+                </page>
+                <page string="Field options"
+                      attrs="{'invisible': [('update_tax', '=', False), ('update_account', '=', False), ('update_fiscal_position', '=', False)]}"
```
Comment (victor-ditay) [inline account_chart_update/wizard/wizard_chart_update_view.xml]: Would be better indent this line if possible

File: account_chart_update/wizard/wizard_chart_update_view.xml [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+                        >
+                            <field name="fp_field_ids"
+                                   widget="many2many_checkboxes"
+                                   context="{'account_chart_update': True}"
+                            />
+                        </page>
+                    </notebook>
+                </page>
+                <page string="Matching"
+                      attrs="{'invisible': [('update_tax', '=', False), ('update_account', '=', False), ('update_fiscal_position', '=', False)]}"
```
Comment (victor-ditay) [inline account_chart_update/wizard/wizard_chart_update_view.xml]: Would be better indent this line if possible

**Review by victor-ditay — APPROVED** [review]
LGTM but could be improved a little bit regarding the readability of the code

**Comment by ernestotejeda** [comment]
@mreficent , @victormmtorres, i did the changes requested, thanks

**Pull request merged**